test/exfmt/integration/map_test.exs

```diff
@@ -47,11 +47,11 @@ defmodule Exfmt.Integration.MapTest do
   end
 
   test "chained map get" do
-    assert_format "map.key.another.a_third\n"
+    "map.key.another.a_third" ~> "map.key().another().a_third()\n"
```

Comment:
Could we change it to not add parens unless the left hand side is an atom or a module alias please? :)